storage/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/Scalingo/go-utils/storage
+
+go 1.14
+
+require (
+	github.com/Scalingo/go-utils/logger v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.35.9
+	github.com/aws/aws-sdk-go-v2 v0.27.0
```

Comment:
I don't understand why we use v1 and v2 of this dependency. Maybe an issue in the code and we should only use v2?